WordPress/src/main/java/org/wordpress/android/ui/posts/PostEditorAnalyticsSession.java

```diff
@@ -148,6 +154,7 @@ public void editorSettingsFetched(Boolean fullSiteEditing, String endpoint) {
         final Map<String, Object> properties = getCommonProperties();
         properties.put(KEY_FULL_SITE_EDITING, fullSiteEditing);
         properties.put(KEY_ENDPOINT, endpoint);
+        properties.put(KEY_ENDPOINT, endpoint);
```

Comment:
Is this duplication intentional, or perhaps an error from resolving a git conflict?

Reply:
Thank you for spotting this Matthew 🙇 
I was thinking of adding the new property on the settings changed event but retracted since this did not align with [iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/16848) and added unnecessary complexity.
I forgot to remove the code duplication as I undid the changes 😞 
Fixed with ec375a093438c09c7b4b46b8edc180a02daac092